LICENSE.txt

```diff
@@ -1 +1 @@
-https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode
\ No newline at end of file
+-https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode
```

Comment:
No need to change this.

Reply:
Yeah I accidentally deleted it for a commit so I just added it back